tests/load/pipeline/test_pipelines.py

```diff
@@ -881,6 +881,8 @@ def some_source():
 def test_pipeline_upfront_tables_two_loads(
     destination_config: DestinationTestConfiguration, replace_strategy: str
 ) -> None:
+    dlt.config["truncate_staging_dataset"] = True
```

Comment:
this is good but we have auto cleanup fixture only for os.environ. so please use it instead